### PR TITLE
Ctda 1600 fix raml

### PR DESCRIPTION
**Pull request by RobertLaverick**
I've two commits, one with and one without a Hateoas response because it slipped through the next last time. I can revert the Hateoas commit if we want to keep it simple

### CODE REVIEW

File: resources/public/api/conf/1.0/application.raml [hunk 1]
```diff
@@ -643,3 +643,23 @@ traits:
                       }
               404:
                 description: When message does not exist, has been archived or is not available to the EORI number.
+  /push-pull-notifications/box:
+    get:
+      displayName: Get Box Info
+      description: |
+        Get Box Information for your application if there is a push-pull-notification box
```
Comment (DavidGregory084) [inline resources/public/api/conf/1.0/application.raml:650]: Does this need content review?

File: app/models/response/HateoasResponseBox.scala [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        .obj(
+          "boxId"                      -> box.boxId,
+          "boxName"                 -> box.boxName,
+          "_links" -> Json.obj(
+            "self"     -> Json.obj("href" -> boxUrl),
+          )
+        )
+        .fields
+        .filter(
+          t => t._2 != JsNull
```
Comment (DavidGregory084) [inline app/models/response/HateoasResponseBox.scala:41]: Can any of these things be null?

**Review by DavidGregory084 — APPROVED** [review]
Looks good to me, just a couple questions that don't necessarily need to block merging to QA

File: app/models/response/HateoasResponseBox.scala (hunk 2 above)
Comment (RobertLaverick) [reply app/models/response/HateoasResponseBox.scala:41]: No, its just a leftover, I'll clean it out.

**Pull request merged**